File: gradle/packages/cli-function/src/main/kotlin/com/app/cli/function/Random.kt
package com.app.cli.function

import com.app.numbers.utils.operations.FactorialUtils.factorialRandom
import com.app.numbers.utils.operations.FibonacciUtils.fibonacciRandom

object Random {
    @JvmStatic
    fun main(args: Array<String>) {
        println("===================================================")
        printRandomNumbers()
        println("===================================================")
    }

    fun printRandomNumbers() {
        println(fibonacciRandom())
        println(factorialRandom())
    }
}

File: cabal/src/CLI/Function/Random.hs
module CLI.Function.Random (printRandomNumbers) where

import Numbers.Utils.Operations.FactorialUtils (factorialRandom)
import Numbers.Utils.Operations.FibonacciUtils (fibonacciRandom)

printRandomNumbers :: IO ()
printRandomNumbers = do
    putStrLn "Some random numbers : "
    putStrLn =<< fibonacciRandom
    putStrLn =<< factorialRandom
